packages/web-features-figuring/src/baseline/support.ts

```diff
@@ -0,0 +1,23 @@
+import { Browser } from "../browser-compat-data/browser";
+import { Compat } from "../browser-compat-data/compat";
+import { Feature } from "../browser-compat-data/feature";
+import { Release } from "../browser-compat-data/release";
+
+type Support = Map<Browser, Release | undefined>;
+
+export function support(feature: Feature, browsers: Browser[], compat: Compat) {
+  const support: Support = new Map();
+  for (const b of browsers) {
+    support.set(b, undefined);
+  }
+
+  for (const rel of feature.supportedBy({ only: browsers, compat })) {
+    const currRel = support.get(rel.browser);
+
+    if (!currRel || currRel.compare(rel) >= 0) {
```

Comment:
What is `currRel` and `rel`? The absence of `compareVersions` anywhere makes me suspect that a bug is on this line, if these are strings.